src/main/resources/css/cucumber.css

```diff
@@ -61,20 +61,24 @@ a:hover {
     overflow: auto;
 }
 
-/* same as <pre> from bootstrap library */
+/*
+   same as <pre> from bootstrap library.
+   padding and overflow-x added.
+*/
 .embedding-content {
-    display: block;
     padding: 10px;
     margin-left: 10px;
     margin-right: 10px;
     margin-bottom: 10px;
     font-size: 13px;
     overflow-x: auto;
     line-height: 1.42857143;
+    color: #333;
+    word-break: break-all;
+    word-wrap: break-word;
```

Comment:
These are in pre from bootstrap. Keeps sections that don't use pre consistent with sections that do.